storage/tokudb/tokudb_sysvars.cc

```diff
@@ -48,41 +48,41 @@ const size_t error_buffer_max_size = 1024;
 ulonglong   cache_size = 0;
 uint        cachetable_pool_threads = 0;
 int         cardinality_scale_percent = 0;
-my_bool     checkpoint_on_flush_logs = FALSE;
+my_bool     checkpoint_on_flush_logs = false;
```

Comment:
false/true are not values of my_bool type, which is char in 5.6. IMHO FALSE/TRUE should be kept for my_bool